src/app/subnets/views/SubnetsList/_index.scss

```diff
@@ -61,4 +61,10 @@
       width: 1px;
     }
   }
+
+  .fabric-table, .space-table {
+    .subnet-row td:first-child {
+      padding-left: 1.8125rem;
```

Comment:
Where does this value come from? I'd normally suggest using one of Vanilla's built-in spacing variables.

Reply:
I tried making the spacing, consistent with the machine list table, but this has been updated